tests/unit/test_data.py

```diff
@@ -68,6 +71,14 @@ def test_len(self, data_type):
         _data = data_type()
         assert len(_data) == len(_data.data)
 
+    @pytest.mark.parametrize("weights", (None, 3, DATA_ARRAY.reshape(-1)))
+    def test_normalize(self, data_type, weights):
```

Comment:
normalise british english?

Reply:
If you search `normalise` it redirects to  https://www.oed.com/dictionary/normalize_v (with a z)

Reply:
What has gone wrong at the OED